db/appsock_handler.c

```diff
@@ -55,6 +55,7 @@ char appsock_unknown_old[] = "-1 #unknown command\n";
 char appsock_unknown[] = "Error: -1 #unknown command\n";
 char appsock_supported[] = "supported\n";
 int active_appsock_conns = 0;
+int64_t gbl_denied_appsock_connection_count = 0;
```

Comment:
since its a counter, can we do uint64_t?

Reply:
This was me matching up definition in `struct comdb2_metrics_store` to resolve compiler warning. I didn't want to change more than what was required.